packages/api/src/name.ts

```diff
@@ -126,6 +127,12 @@ export async function namePost (request: Request, env: Env, ctx: ExecutionContex
 
     try {
       const objPostResponse: Response = await obj.fetch(postRequest)
+      const ipnsRecord: IPNSRecordDataWithMeta = await objPostResponse.json()
```

Comment:
Could we use the HTTP status (200 and 201) instead of adding a new field?

Reply:
Great suggestion. Done.